notebooks/.gitignore

```diff
@@ -0,0 +1 @@
+.*
```

Comment:
Is a second .gitignore file needed?
In the original, we could do
`/notebooks/.*`